packages/playkit-js-scripts/scripts/serve.js

```
@@ -0,0 +1,72 @@
+const commander = require('commander');
+const path = require('path');
+const spawn = require('cross-spawn');
+const fs = require('fs-extra');
+const prompts = require('prompts');
+const os = require('os');
+
+const modes = {
+    userType: ['annonymous', 'widgetId', 'ks'],
+    bundler: ['custom', 'uiconf'],
+    serverEnv: ['qa', 'production'],
+    bundlerEnv: ['qa', 'production'],
+};
+
+const modesTypes = Object.keys(modes);
+
+async function modeSelection() {
+    const onCancel = prompt => {
+        console.log(`${chalk.red('Canceled!')}`);
+        process.exit(1);
+    };
+
+    const chooseModes = modesTypes.map(mode => ({
+        type: "select",
+        name: mode,
+        message: `Choose ${mode}:`,
+        choices: modes[mode].map(value => ({title: value, value})),
+        initial: 0,
+    }));
+
+    return await prompts(chooseModes, {onCancel});
+}
+
+(async () => {
+    const appPath = process.cwd();
+    const appTestFolder = `${appPath}/test`;
+    const appConfigPath = path.join(appTestFolder, 'config.json');
+    const appEnvPath = path.join(appTestFolder, 'env.json');
+    let isFilesExist = false;
+
+    try {
+        isFilesExist = [appConfigPath, appEnvPath].map(file => fs.existsSync(file)).every(Boolean);
+    } catch (e) {
+        console.error('Error while reading files paths.');
```

is it desired that errors are ignored silently instead of stoping the process?